extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json

```diff
@@ -63,9 +70,14 @@
           "meta:descriptionId": "message-interaction##https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/label##description##5891"
         },
         "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/offers": {
-          "title": "Offer details",
+          "title": "Offer Details",
           "$ref": "https://ns.adobe.com/experience/customerJourneyManagement/offers",
           "description": "Describes offer(s) involved in a message interaction."
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/subscription": {
+          "title": "Subscription Details",
+          "$ref": "https://ns.adobe.com/experience/customerJourneyManagement/subscription",
```

Comment:
If there are no intentions of re-using that new subscription datatype, then simply nest the fields under this parent "subscription" field. Only create a datatype if you have multiple fields that need to be of that type.